src/test/kotlin/org/rust/lang/refactoring/RsImportOptimizerTest.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.refactoring
+
+import com.intellij.openapi.actionSystem.IdeActions
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+
+class RsImportOptimizerTest: RsTestBase() {
+
+    fun `test should do nothing`() = doTest("""
+        use foo;
+    """, """
+        use foo;
+    """)
+
+    fun `test should be at the beginnging`() = doTest("""
+        //! test
+        extern crate log;
+
+        fn test() {}
+        use foo;
+    """, """
+        //! test
+        extern crate log;
+
+        use foo;
+
+        fn test() {}
+
+    """)
+
+    fun `test sort alphabetical of useItem`() = doTest("""
+        use foo;
+        use bar;
+    """, """
+        use bar;
+        use foo;
+
+    """)
```

Comment:
@Undin Any tip to fix this hack?

Reply:
I don't really know how to avoid extra line after element deletion at last line of a file.
Let's leave it as is